prepackPy/splitter.py

```diff
@@ -1,33 +1,94 @@
 #!/usr/bin/env python
 
 import numpy as np
-import warnings
 import pandas as pd
 
+
 def splitter(X, target_index, split_size, seed): 
     """
     split a dataset into train and test sets
 
     Parameters
     ----------
-    X: numpy array, pandas dataframe
-        an input dataset
+    X: numpy ndarray, pandas dataframe
+        input dataset
     target_index: integer
-        a column index of target variable in X
+        column index of target variable in X
     split_size: float
-        a proportion of the dataset to include in the test split
+        proportion of the dataset to include in the test split
     seed: integer
-        a random state that will make code reproducible
+        random state that will make code reproducible
 
     Returns
     -------
-    X_train: numpy array
+    X_train: numpy ndarray
         splitted features for model training
```

Comment:
I'm not sure 'splitted' is correct grammar.  Maybe "split features for model training"?

Reply:
Ok, I will fix them.